### PR TITLE
Release v24.6.1

### PR DESCRIPTION
**Pull request by mikemckiernan**
RN: https://nvidia.github.io/cloud-native-docs/review/pr-87/gpu-operator/latest/release-notes.html

Matrix: https://nvidia.github.io/cloud-native-docs/review/pr-87/gpu-operator/latest/platform-support.html#gpu-operator-component-matrix

### CODE REVIEW

File: gpu-operator/life-cycle-policy.rst [hunk 1]
```diff
@@ -99,7 +99,7 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
      - `1.16.1 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
 
    * - NVIDIA Kubernetes Device Plugin
-     - `0.16.1 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
+     - `0.16.2 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
```
Comment (cdesiniotis) [inline gpu-operator/life-cycle-policy.rst:102]: We also need to bump the operator / validator versions in this table to 24.6.1

File: gpu-operator/release-notes.rst [hunk 2]
```diff
@@ -34,6 +34,30 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
...
+======
+
+.. _v24.6.1-new-features:
+
+New Features
+------------
+
+* Added support for NVIDIA Container Toolkit v1.16.2.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
```
Comment (cdesiniotis) [inline gpu-operator/release-notes.rst:49]:
```suggestion
* Added support for NVIDIA Kubernetes Device Plugin v0.16.2.
  Refer to the :ref:`GPU Operator Component Matrix`
  on the platform support page.
```

File: gpu-operator/release-notes.rst [hunk 3]
```diff
@@ -34,6 +34,36 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
...
+======
+
+.. _v24.6.1-new-features:
+
+New Features
+------------
+
+* Added support for the following software component versions:
+
+  - NVIDIA Container Toolkit v1.16.2
```
Comment (cdesiniotis) [inline gpu-operator/release-notes.rst:49]: We did not bump the toolkit version
```suggestion
```

Comment (mikemckiernan) [reply gpu-operator/release-notes.rst:49]: TY.  I meant to add ` NVIDIA GPU Feature Discovery for Kubernetes v0.16.2`.

File: gpu-operator/release-notes.rst [hunk 4]
```diff
@@ -34,6 +34,36 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
...
+  on the platform support page.
+
+.. _v24.6.1-fixed-issues:
+
+Fixed Issues
+------------
+
+* Fixed an issue with role-based access controls that prevented a service account from accessing config maps.
+  Refer to Github `issue #883 <https://github.com/NVIDIA/gpu-operator/issues/883>`__ for more details.
+* Fixed an issue with role-based access controls that prevented retrieving NVIDIA Driver daemon set information.
```
Comment (tariq1890) [inline gpu-operator/release-notes.rst:62]: Can we specify that this was a fix in `gpu-operator-validator`?

**Review by tariq1890 — APPROVED** [review]

**Review by cdesiniotis — APPROVED** [review]

**Pull request merged**